jetty-server/src/main/java/org/eclipse/jetty/server/handler/gzip/GzipHandler.java

```diff
@@ -616,6 +616,9 @@ public void setInflateBufferSize(int size)
     @Override
     public void handle(String target, Request baseRequest, HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException
     {
+        if (response.isCommitted() || baseRequest.isHandled())
```

Comment:
I know we do sometimes check isCommitted elsewhere, but I think isHandled is enough here.

Reply:
but if the request is handled, do we even want to?